js/src/wp-seo-admin-global.js

```diff
@@ -374,6 +449,8 @@
 				$link.parent().addClass( "active" );
 
 				openHelpCenterTab( $container, $( "#" + target ) );
+				// Trigger a custom event to check the scrollable tables size.
+				$( window ).trigger( "yoast-table-scrollable-check-size" );
```

Comment:
Doesn't this mean the event gets triggered twice whenever there are active link tabs?